tests/test_eda_analysis.py

```diff
@@ -1 +1,88 @@
-from eda_analysis import eda_analysis
+from eda_analysis import eda_analysis as eda
+import numpy as np
+import pandas as pd
+import altair as alt
+import pytest
+
+
+
+def helper_create_data(n = 500):
+    """
+    Helper function for creating dataframe for testing
+    
+    Parameters:
+    -----------
+    n: int (default value = 500)
+        Number of rows to be generated for the dataframe
+    
+    Returns:
+    --------
+    pandas.DataFrame
+        Returns a dataframe to be used for testing
+        
+    Examples:
+    ---------
+    >>> helper_create_data()
+    """
+    N1 = list(np.random.exponential(3,n))
+    N2 = list(np.random.normal(2,2,n))
+    N3 = list(np.random.normal(10,3,n))
+    C1 = list(np.random.binomial(1,0.7,n))
+    C2 = list(np.random.poisson(1,n))
+    C3 = list(np.random.binomial(5,0.4,n))
+    a = ['cat','dog','lion']
+    C4 = list(np.random.choice(a,n))
+    df = pd.DataFrame({
+        'C1':C1,
+        'C2':C2,
+        'C3':C3,
+        'N1':N1,
+        'N2':N2,
+        'N3':N3, 
+        'C4':C4
+    })
+    
+    rows = list(np.random.randint(0,n,20))
+    cols = list(np.random.randint(0,7,5))
+    df.iloc[rows,cols] = np.nan
+    
+    return df
+
+
```

Comment:
The test function looks great. 
Need to add tests accordingly if you do not use the melt dataframe.
And add tests accordingly if you make some changes. For example, the test for the `n_cols` input.

Reply:
Only need to add tests if you add the `n_col` checking in your main function.

Reply:
added test for n_col